desktop/register.js

```diff
@@ -0,0 +1,6 @@
+import {XH} from '@xh/hoist/core';
+
+if(XH._mobileAPIRegistered){
```

Comment:
fine, but we should use `XH.exception()` to create this exception for good measure.